journal-of-korean-neurosurgical-society.csl

```diff
@@ -15,7 +15,6 @@
     <category field="medicine"/>
     <issn>2005-3711</issn>
     <eissn>1598-7876</eissn>
-    <summary>A style for the Journal of Korean Neurosurgical Society based on ElesvierVancouver/Free Radical Research.</summary>
```

Comment:
This wasn't right, right? (the template is "american-journal-of-translational-research")